pyproject.toml

```
@@ -50,7 +50,10 @@ dev = [
     "pre-commit", 
     "ruff"
 ]
-cartopy = ["cartopy>=0.16"]
+geomap = [
```

#912 drops requests as an dependency, so I wouldn't rename it
```suggestion
cartopy = [
```